docs/sources/datasources/azuremonitor.md

```diff
@@ -275,10 +277,15 @@ Example variable queries:
 | `workspaces()`                                                                          | Returns a list of workspaces for default subscription     |
 | `workspaces("12345678-aaaa-bbbb-cccc-123456789aaa")`                                    | Returns a list of workspaces for a specified subscription |
 | `workspaces("$subscription")`                                                           | With template variable for the subscription parameter     |
+| `resources()`                                                                           | Returns a list of resources for default subscription      |
+| `resources("12345678-aaaa-bbbb-cccc-123456789aaa")`                                     | Returns a list of resources for a specified subscription  |
+| `resources("$subscription")`                                                            | With template variable for the subscription parameter     |
 | `workspace("myWorkspace").Heartbeat \| distinct Computer`                               | Returns a list of Virtual Machines                        |
 | `workspace("$workspace").Heartbeat \| distinct Computer`                                | Returns a list of Virtual Machines with template variable |
 | `workspace("$workspace").Perf \| distinct ObjectName`                                   | Returns a list of objects from the Perf table             |
 | `workspace("$workspace").Perf \| where ObjectName == "$object" \| distinct CounterName` | Returns a list of metric names from the Perf table        |
+| `resource("myResource").Heartbeat \| distinct Computer`                                 | Returns a list of Virtual Machines                        |
```

Comment:
```suggestion
| `resource("myResource").Heartbeat \| distinct Computer`                                 | Returns a list of virtual machines                        |
```

Reply:
Resolving these suggestions to be consistent with the description above.